_includes/downloads-scala2.html

```diff
@@ -1,70 +1,38 @@
-<h2>The most popular way to get Scala is either using Scala through sbt, the Scala build tool, or to use Scala through an IDE.</h2>
 <div class="install-steps">
+  <h3>Release Notes</h3>
+  For a summary of important changes, see the <a href="https://github.com/scala/scala/releases">GitHub release notes</a>.
+  <br /><small>(Or consult our archive of <a href="{{ site.baseurl }}/blog/announcements/">older release
+      notes</a>.)</small>
+
   <div class="step arrow">
     <img src="/resources/img/download/arrow-asset.png" alt="">
-    <div class="number-step">1</div>
-    <div class="text-step">
-      <h3>First, make sure you have the Java 8 JDK (or Java 11 JDK) installed.</h3>
-      <p>To check, open the terminal and type:</p>
-      <p><code>java -version</code><span>(Make sure you have version 1.8 or 11.)</span></p>
-      <p><i>(If you don't have it installed, download Java from <a href="https://www.oracle.com/java/technologies/javase-jdk8-downloads.html">Oracle Java 8</a>, <a href="https://www.oracle.com/java/technologies/javase-jdk11-downloads.html">Oracle Java 11</a>, or <a href="https://adoptopenjdk.net/">AdoptOpenJDK 8/11</a>. Refer <a href="https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html">JDK Compatibility</a> for Scala/Java compatiblity detail.</i></p>
-    </div>
-  </div>
-  <div class="step">
-    <div class="number-step">2</div>
+    <div class="number-step"><i class="fa fa-check" aria-hidden="true"></i></div>
     <div class="text-step">
-      <h3>Then, install Scala:</h3>
-      <p>...either by installing an IDE such as IntelliJ, or sbt, Scala's build tool.</p>
-    </div>
-  </div>
-  <div class="download-options">
-    <div class="download-left">
-      <span class="or">or</span>
-      <div class="description">
-        <img src="/resources/img/download/arrow-left.png" alt="">
-        <p>Best if you prefer a full-featured IDE (recommended for beginners)</p>
+      <h2>Recommended Download</h2>
+      <div class="download-options">
+        <a href="https://docs.scala-lang.org/getting-started/index.html" class="btn-download" id="get-started-link">
+          <i class="fa fa-download"></i>
+          <span>Get Started With Scala</span>
```

Comment:
Users came here by following the link “pick a specific release”, which is labeled “best if you have an advanced use case”. I don’t think we should point them back to the general getting started instructions.

Reply:
removed in next commit